streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -998,6 +999,23 @@ public void shouldUseSpecifiedNameForAggregateOperationGivenTable() {
             STREAM_OPERATION_NAME);
     }
 
+    @Test
+    public void shouldUseSpecifiedNameForGlobalStoreProcessor() {
```

Comment:
I didn't see there is tests for `GlobalStoreProcessor` using default name, right? That is, a test to create global store by `addGlobalStore` with `Consumed` instance without specifying a custom name, and verify the names.